venv/task1.py

```diff
@@ -0,0 +1,55 @@
+import connections as con
+
+newDF = con.titles_info.join(con.rating_info,
+                con.titles_info.tconst == con.rating_info.tconst.alias('tconst'), 'inner') \
+        .withColumn('numVotes', con.rating_info.numVotes.cast('integer'))\
+        .drop(con.titles_info.tconst)\
+
+
+def TopFilmsForAllTimes():
+    topFilmsForAllTimes = newDF.select(
+                newDF.tconst, newDF.primaryTitle,
+                newDF.numVotes, newDF.averageRating, newDF.startYear) \
+        .where((newDF.numVotes >= 100000)
+               & (newDF.titleType == 'movie')) \
+        .orderBy(newDF.averageRating.desc(), newDF.numVotes.desc()) \
+        .limit(100)
+
+    topFilmsForAllTimes.write.format('csv')\
+        .option("header", True).mode('overwrite')\
+        .save("D:\pythonProject1\outputs\TopFilmsForAllTimes")
```

Comment:
I think this functionality should be moved to a separate function? 
you reuse this functionality already at least 3 times for now